src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -10,12 +10,12 @@
 Copyright Contributors to the Zowe Project.
 """
 
-import os
 
 from zowe.core_for_zowe_sdk import SdkApi
-from zowe.core_for_zowe_sdk.exceptions import FileNotFound
-from zowe.zos_files_for_zowe_sdk import constants, exceptions
-from zowe.zos_files_for_zowe_sdk.constants import FileType, zos_file_constants
+from zowe.zos_files_for_zowe_sdk.constants import FileType
+from .datasets import Datasets
+from .uss import USSFiles
+from .file_system import FileSystems
 
 _ZOWE_FILES_DEFAULT_ENCODING = "utf-8"
```

Comment:
I know all functions in this class are deprecated, but curious if we should also move this into a `constansts.py` file.

Reply:
I'm not sure as I hardly think these imports would be used again in another file.